### PR TITLE
Add security analytics plugin

### PR DESCRIPTION
**Pull request by JavierBejMen**
|Related issue|
|---|
|#20090|

This PR adds the OpenSearch security analytics plugin installation on the Vagrant provision script. Also improves windows decoders integrity of the schema field mappings.

### CODE REVIEW

File: src/engine/ruleset/integrations/windows/decoders/windows-security.yml [hunk 1]
```diff
@@ -167,11 +167,11 @@ normalize:
   # Group: Target User 1 - Set Target User Domain
   - check: array_contains($_ev_target_user_1, $event.id) AND not_exists($user.domain)
     map:
-      - user.domain: $windows.EventData.TargetDomainName
+      - user.domain: parse_fqdn($windows.EventData.TargetDomainName)
```
Comment (soynof) [inline src/engine/ruleset/integrations/windows/decoders/windows-security.yml:170]: The `parse_fqdn($windows.EventData.TargetDomainName)` statement is used multiple times, would it be better to create a variable to assign it only once and execute the hf `parse_fqdn` only once? what do you think?

**Review by soynof — CHANGES_REQUESTED** [review]

**Review by soynof — APPROVED** [review]
LGTM!

**Review by juliancnn — APPROVED** [review]
LGTM!

**Review by JcabreraC — APPROVED** [review]
LGTM !

**Pull request merged**